robottelo/api/utils.py

```diff
@@ -293,7 +293,7 @@ def configure_provisioning(org=None, loc=None, compute=False, os=None):
     # Create a Product, Repository for custom RHEL6 contents
     product = entities.Product(organization=org).create()
     repo = entities.Repository(
-        product=product, url=settings.rhel7_os, download_policy='immediate'
+        product=product, url=str(settings.rhel7_os), download_policy='immediate'
```

Comment:
I think it would be better to add `rhel7_os` to wrapper exceptions list in [robottelo/config/facade.py](https://github.com/SatelliteQE/robottelo/blob/master/robottelo/config/facade.py). I see `rhel8_os` is already there.